python_cmake_module/cmake/Modules/FindPythonExtra.cmake

```diff
@@ -211,7 +222,11 @@ if(PYTHONINTERP_FOUND)
   endif()
 
   if(WIN32)
-    set(PythonExtra_EXTENSION_EXTENSION ".pyd")
+    if("${CMAKE_BUILD_TYPE} " STREQUAL "Debug ")
+      set(PythonExtra_EXTENSION_EXTENSION "_d.pyd")
```

Comment:
PythonExtra now performs the existence test of the debug interpreter and throw an error if unable to find it. https://github.com/ros2/rosidl/pull/153/commits/52eb0f4e9d76ddd7d3dfd8f5f39f3cdb764c55d2